src/sieppari/context.cljc

```diff
@@ -1,17 +1,20 @@
 (ns sieppari.context
   (:require [sieppari.queue :as q])
-  (:import (clojure.lang PersistentQueue)))
+  #?(:clj
+     (:import (clojure.lang PersistentQueue))))
 
 (defn terminate
   "Removes all remaining interceptors from context's execution queue.
   This effectively short-circuits execution of Interceptors' :enter
   functions and begins executing the :leave functions.
   Two arity version allows setting the response at the same call."
   ([ctx]
-   (assoc ctx :queue PersistentQueue/EMPTY))
+   (assoc ctx :queue #?(:clj PersistentQueue/EMPTY
+                        :cljs #queue [])))
   ([ctx response]
    (-> ctx
-       (assoc :queue PersistentQueue/EMPTY)
+       (assoc :queue #?(:clj PersistentQueue/EMPTY
+                        :cljs #queue []))
```

Comment:
Could use `q/empty-queue` here if it was public.

Reply:
I can make it public, definitely